src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -92,6 +92,9 @@ protected function describeRoute(Route $route, array $options = array())
             array('Defaults', $this->formatRouterConfig($route->getDefaults())),
             array('Options', $this->formatRouterConfig($route->getOptions())),
         );
+        if ($options['callable']) {
```

Comment:
Looking at the build failures, you need to check first that the `callable` key does exist.